src/firecracker/src/main.rs

```diff
@@ -399,18 +389,6 @@ fn generic_error_exit(msg: &str) -> ExitCode {
     vmm::FC_EXIT_CODE_GENERIC_ERROR
 }
 
-// Log a warning for any usage of deprecated parameters.
-fn warn_deprecated_parameters(arguments: &Arguments) {
-    // --seccomp-level is deprecated.
-    if let Some(value) = arguments.single_value("seccomp-level") {
-        warn!(
-            "You are using a deprecated parameter: --seccomp-level {}, that will be removed in a \
-            future version.",
-            value
-        );
-    }
-}
```

Comment:
This function is part of a step in deprecating command line parameters, link to API change runbook [here](https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md#marking-endpoints-as-deprecated). Should we just leave an empty body with #[allow(unused)] as we may add something here in the future? WDYT

Reply:
Given that it's explicitly mentioned in the API change runbook, I think it makes sense keeping it empty, will do that

Reply:
done